Source/WebCore/Modules/webcodecs/WebCodecsVideoEncoder.cpp

```diff
@@ -67,10 +67,17 @@ WebCodecsVideoEncoder::~WebCodecsVideoEncoder()
 {
 }
 
-static bool isValidEncoderConfig(const WebCodecsVideoEncoderConfig& config, const Settings::Values& settings)
+static bool isSupportedEncoderCodec(const String& codec, const Settings::Values& settings)
 {
-    // FIXME: Check codec more accurately.
```

Comment:
the new test is identical, just reformatted. This FIXME no longer applies?

Reply:
Codec specific checks are now resolved asynchronously.
We move them closer to each encoder, which might parse/validate the profile information.
We have done so for decoders already.